c4_build_catalog.m

```diff
@@ -0,0 +1,44 @@
+
+cooksey_c4_catalog = fitsread('/home/reza/gp/c4/Cooksey_c4_cat/c4_catalogs/distfiles/Cooksey_C4_detected.fits', 'binarytable');
+c4_QSO_ID = cooksey_c4_catalog{1};
+c4_NCIV = cooksey_c4_catalog{10};
+c4_zCIV = cooksey_c4_catalog{3};
+f = fopen('los_catalog','w');
+for i=1:size(c4_zCIV)
+    
+    fprintf(f,'%s \n', c4_QSO_ID{i});
+
+end
+
+f = fopen('c4_catalog','w');
+
+for i=1:size(c4_zCIV)
+    
+    fprintf(f,'%s  %f %f\n', c4_QSO_ID{i}, c4_zCIV(i), c4_NCIV(i));
+
+end
+% There are some NAN valued c4_NCIV
+
+% extract basic QSO information from Cookse_all_QSO catalog 
+cooksey_all_qso_catalog =  fitsread('/home/reza/gp/c4/Cooksey_c4_cat/c4_catalogs/distfiles/Cooksey_all_QSO.fits', 'binarytable');
```

Comment:
Also avoid hard-code the path in this line

Reply:
Done!